rs/boundary_node/prober/src/main.rs

```diff
@@ -575,7 +575,7 @@ impl Install for Installer {
             0,
         );
 
-        install_call
+        () = install_call
             .call_and_wait()
             .await
             .context("failed to install canister")?;
```

Comment:
Just curious: what kind of lint is this? Because at first sight it looks rather stupid.

Reply:
It will turn into a hard error in Rust 2024

Here's the tracking issue for the lint:
- https://github.com/rust-lang/rust/issues/123748

And here is a WIP rust edition guide with an explaination:
- https://doc.rust-lang.org/nightly/edition-guide/rust-2024/never-type-fallback.html

Reply:
Very interesting, thanks for the links

Reply:
Wow, that's obscure.